web-api/src/business/useCases/caseDeadline/createCaseDeadlineInteractor.ts

```diff
@@ -33,13 +33,14 @@ export const createCaseDeadline = async (
     associatedJudgeId: caseEntity.associatedJudgeId,
   });
 
-  await upsertCaseDeadlines([newCaseDeadline.validate().toRawObject()]);
+  const caseDeadlines = await upsertCaseDeadlines([newCaseDeadline.validate().toRawObject()]);
 
   caseEntity = await applicationContext
     .getUseCaseHelpers()
     .updateCaseAutomaticBlock({
       applicationContext,
       caseEntity,
+      deadlines: caseDeadlines
```

Comment:
NOTE: This _isn't_ all of the case deadlines, but it still works because of how updateAutomaticBlocked works (it just checks for empty). This is a bit more opaque than I would like. Can we modify updateAutomaticBlocked to take in a more sensible parameter since we don't actually _use_ the case deadlines except for the isEmpty check?

Reply:
> This is a bit more opaque than I would like
Because now a future developer might make changes under the reasonable assumption that they have all case deadlines in updateAutomaticBlocked ... but they don't!